bodo/io/parquet_write.cpp

```diff
@@ -253,7 +253,7 @@ int64_t pq_write(const char *_path_name,
         // This is ugly and should be moved to python, abfs_get_fs
         // when we upgrade to Arrow 20.
         std::regex r(".*[?&]sv=.*&sig=.*");
-        regex_search(path_name, match, r);
+        regex_search(orig_path, match, r);
```

Comment:
So path name is being converted to just the file path from my PR. We are searching for the SAS token which is in the original path. So just need to update this. Seemed to work